sdk/go/http/http.go

```diff
@@ -34,6 +34,26 @@ func NewRouter() *Router {
 	return httprouter.New()
 }
 
+// NewTransport returns spin-go sdk backed http.RoundTripper
+func NewTransport() http.RoundTripper {
+	return &Transport{}
+}
+
+// Transport implements http.RoundTripper
+type Transport struct{}
+
+// RoundTrip makes roundtrip using spin-go sdk
+func (r *Transport) RoundTrip(req *http.Request) (*http.Response, error) {
+	return Send(req)
+}
+
+// NewClient returns spin-go sdk backed *http.Client
```

Comment:
nit: we never refer to the Go SDK as `spin-go`.
suggestion: `NewClient returns a new HTTP client compatible with the Spin SDK` maybe?

Reply:
done